rero_ils/modules/stats_cfg/api.py

```diff
@@ -59,6 +59,13 @@ class StatConfiguration(IlsRecord):
     provider = StatCfgProvider
     model_cls = StatCfgMetadata
 
+    def get_active_stats_configurations():
```

Comment:
We decided to place this kind of ES query directly into the `Search` class (here StatsConfigurationSearch)